diff --git a/pywren/scripts/pywrencli.py b/pywren/scripts/pywrencli.py
index b8609fb..f633824 100644
--- a/pywren/scripts/pywrencli.py
+++ b/pywren/scripts/pywrencli.py
@@ -328,7 +328,7 @@ def deploy_lambda(ctx, update_if_exists=True):
 
                 lambclient.create_function(FunctionName=FUNCTION_NAME,
                                            Handler=pywren.wrenconfig.AWS_LAMBDA_HANDLER_NAME,
-                                           Runtime="python2.7",
+                                           Runtime="python3.7",
                                            MemorySize=MEMORY,
                                            Timeout=TIMEOUT,
                                            Role=ROLE,
diff --git a/pywren/wrenhandler.py b/pywren/wrenhandler.py
index 016c5d3..fd3a2f6 100644
--- a/pywren/wrenhandler.py
+++ b/pywren/wrenhandler.py
@@ -35,8 +35,12 @@ import botocore
 
 if sys.version_info > (3, 0):
     from queue import Queue, Empty # pylint: disable=import-error
-    from . import wrenutil # pylint: disable=relative-import
-    from . import version  # pylint: disable=relative-import
+    try:
+        from . import wrenutil # pylint: disable=relative-import
+        from . import version  # pylint: disable=relative-import
+    except (ImportError, ModuleNotFoundError) as e:
+        import wrenutil # pylint: disable=relative-import
+        import version  # pylint: disable=relative-import
 
 else:
     from Queue import Queue, Empty # pylint: disable=import-error
